pom.xml

```diff
@@ -400,9 +436,18 @@
 
         <testng.version>6.9.10</testng.version>
         <jacoco.version>0.7.9</jacoco.version>
-        <powermock.version>1.7.3</powermock.version>
+        <powermock.version>1.6.5</powermock.version>
```

Comment:
why downgrade?

Reply:
Earlier I got some problem with running the test cases.Now I 'll try to restore it